.gitlab-ci.yml

```diff
@@ -38,6 +38,25 @@ default:
         - cd $CORE_BUILD_DIR
         - make -j check
 
+.test-pmix:
+    extends: .lc-variables
+    variables:
+        PYTHON: "/usr/bin/python3"
+        debug: t
+        FLUX_TESTS_LOGFILE: t
+    script:
+        - !reference ['.build-core', 'script']
+        - cd $CORE_BUILD_DIR
+        - export PKG_CONFIG_PATH=$(pwd)/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
```

Comment:
You already have the `$CORE_BUILD_DIR` in a variable, so the `cd $CORE_BUILD_DIR` and then use of `$(pwd)` is not necessary. You could avoid changing the working directory of the script by just using `$CORE_BUILD_DIR`, e.g.:
```
- export PKG_CONFIG_PATH=${CORE_BUILD_DIR}/lib/pkgconfig:$(pkg-config --variable pc_path pkg-config)
```

It could also be confusing that `CORE_BUILD_DIR` refers to the install prefix. Perhaps the CORE_BUILD_DIR and CORE_INSTALL_PREFIX should be different?